test/functional/kernel/nested-loop/tests/nested-loop-Basic-impl.hpp

```diff
@@ -146,6 +149,19 @@ void KernelNestedLoopTest(const DEPTH_3&,
                                        test_array);
 }
 
+// DEPTH_3_COLLAPSE and DEVICE_DEPTH_3 execution policies use the above DEPTH_3 test.
+template <typename WORKING_RES, typename EXEC_POLICY, bool USE_RESOURCE, typename... Args>
+void KernelNestedLoopTest(const DEPTH_3_COLLAPSE&, Args... args){
+  KernelNestedLoopTest<WORKING_RES, EXEC_POLICY, USE_RESOURCE>(DEPTH_3(), args...);
+}
```

Comment:
Please add a blank line after line 156, 160, and 164. Whitespace helps with readability.

Reply:
Done